src/test/java/ibm/jceplus/junit/openjceplus/integration/TestTLS.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright IBM Corp. 2024
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution.
+ */
+package ibm.jceplus.junit.openjceplus.integration;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import ibm.jceplus.junit.base.integration.BaseTestTLS;
+
+public class TestTLS extends BaseTestTLS {
+
+    @BeforeAll
+    public static void init() throws Exception{
+        if (java.security.Security.getProvider("OpenJCEPlusFIPS") != null) {
+            java.security.Security.removeProvider("OpenJCEPlusFIPS");
+        }
+        insertProvider("OpenJCEPlus", "com.ibm.crypto.plus.provider.OpenJCEPlus", 1);
```

Comment:
Add a check to ensure the openjceplusfips provider is not included when running the openjceplus integration test.